src/index.html

```diff
@@ -5,8 +5,417 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__img">
+        <img
+
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="#link1" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#link2" class="nav__link">
+            Samsung
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#link3" class="nav__link">
+            Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#link4"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#link5" class="nav__link">
+            Гаджеты
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#link6" class="nav__link">
+            Планшеты
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#link7" class="nav__link">
+            Фото
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#link8" class="nav__link">
+            Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+    <main class="catalog">
+      <div class="catalog__cards">
+
+        <div class="cards catalog__card" data-qa="card">
+          <img
+            class="cards__img"
+            src="images\imac.jpeg"
+            alt="imac_photo"
+          >
+
+          <p class="cards__text">
+            Моноблок APPLE A1419 iMac 27&quot Retina 5K (MNED2UA/A)
+          </p>
+
+          <p class="cards__code">
+            Код товара: 195434
+          </p>
+
+          <div class="cards__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="cards_review-counter">
```

Comment:
Rename in all cards.
```suggestion
            <p class="cards__review-counter">
```